builder-integration.ts

```diff
@@ -751,9 +789,11 @@ export function builderDevTools(opts: BuilderioOptions = {}): Plugin {
         req.socket.address();
 
         const orgResponseEnd = res.end;
-        res.end = function (...args: any[]) {
+        res.end = function (...args: unknown[]) {
```

Comment:
question: Why are you not typing args the same way as you did on line 811?

Reply:
Because, I didn't write this file.